docs/Dependencies.md

```diff
@@ -7,36 +7,42 @@
 2. **YARP:** Using superbuild install the [YARP](http://www.yarp.it/): to handle the comunication with the robot with both ovrheadset and SDLjoypad drivers;
 
    + if you are not using superbuild, look at [this link](http://www.yarp.it/) for installing YARP.
+   + Use YARP and icub repos using the devel branch instead of the master branch. The main reason is beacuase of the `camCalibWithPose` application.
```

Comment:
```suggestion
   + Use YARP and icub repos using the devel branch instead of the master branch. The main reason is because of the `camCalibWithPose` application.
```